doc/sr/weechat_plugin_api.sr.adoc

```diff
@@ -1538,7 +1538,7 @@ char **weechat_string_split (const char *string, const char *separators,
 * _separators_: граничници који се користе за поделу
 * _strip_items_: карактери који се одсецају од ставки које се враћају (лево/десно); није обавезном, може бити NULL
 * _flags_: комбинација вредности која мења подразумевано понашање; ако је вредност 0, користи се подразумевано понашање (нема одсецања граничника са почетка/краја стринга, вишеструки граничници се остављају какви јесу, па могу да се врате и празни стрингови); прихватају се следеће заставице:
-** WEECHAT_STRING_SPLIT_STRIP_LEFT: граничници се одсецају с лева (почетка стринга)
+** WEECHAT_STRING_SPLIT_STRIP_LEFT: граничници се одсецају с лева (почетка стринга) 
```

Comment:
Hi,
I fixed and removed some trailing whitespace in a previous commit, please don't reintroduce them.

Reply:
Sorry for that, I'll take care in the future.